src/frontend/e2e/utils/general.ts

```diff
@@ -1,4 +1,4 @@
-import { faker } from "@faker-js/faker";
+const faker = require("@faker-js/faker");
```

Comment:
oh i see why you added `@typescript-eslint/no-var-requires` is there a reason why we're doing this syntax and not the above?